src/tasks/task-bholdus-bsc/services/bholdus-header-writer.service.ts

```diff
@@ -11,6 +14,15 @@ export class BholdusHeaderWriterService implements IService<BholdusBscTaskBus> {
 	constructor() {}
 
 	async start(): Promise<void> {
-		const api = await ApiPromise.create(getApiOptions());
+		// const api = await ApiPromise.create(getApiOptions());
+		const wsProvider = new WsProvider('ws://127.0.0.1:9944');
+		const api = await ApiPromise.create({ provider: wsProvider });
+
+		// Do something
+		this.bus.channel(BholdusWriterMessage).subscribe({next:(message: BscPrimitivesBscHeader[]) =>{
+			let req = api.tx.bsc.verifyAndUpdateAuthoritySetSigned(message)
```

Comment:
We need to have a relayer account on Bholdus so it can sign transaction. Is this tested?

Reply:
Please read this https://polkadot.js.org/docs/api/start/api.tx